apps/api/src/server/utils/mapping/map-project-team.js

```diff
@@ -0,0 +1,19 @@
+import { pick } from 'lodash-es';
+
+/**
+ *
+ * @param {import('@pins/applications.api').Schema.ProjectTeam} teamMember
+ * @returns {{role: string, userId: string}}
+ */
+export const mapProjectTeamMember = (teamMember) => {
+	return pick(teamMember, ['role', 'userId', 'createdAt']);
+};
+
+/**
+ *
+ * @param {import('@pins/applications.api').Schema.ProjectTeam[]} teamMembers
+ * @returns {{role: string, userId: string}[]}
+ */
+export const mapProjectTeamMembers = (teamMembers) => {
+	return teamMembers?.map((teamMember) => mapProjectTeamMember(teamMember));
```

Comment:
This will return undefined if `teamMembers` is undefined, which doesn't match the return type. (Also may as well just pass the function directly to map rather than using a lambda.)

```suggestion
	return teamMembers?.map(mapProjectTeamMember) ?? [];
```